src/Aquarius.ONE.ClientSDK/ClientSDK.cs

```diff
@@ -9,6 +9,7 @@
 using ONE.Enterprise.Core;
 using ONE.Enterprise.Report;
 using ONE.Enterprise.Twin;
+using ONE.Operations.Sample;

```

Comment:
I don't think this using needs to be added, I viewed the full file and it already exists.